src/main/java/org/synyx/urlaubsverwaltung/person/web/PersonDetailsBasedataDto.java

```diff
@@ -0,0 +1,20 @@
+package org.synyx.urlaubsverwaltung.person.web;
+
+public class PersonDetailsBasedataDto {
```

Comment:
why is this in another package? Put it next to the controller :)

Reply:
This Dto is used in `PersonDetailsViewController` which resides in the same package.